handler/src/main/java/io/netty/handler/ssl/SslHandler.java

```diff
@@ -429,6 +429,10 @@ public void handlerRemoved0(ChannelHandlerContext ctx) throws Exception {
             // Check if queue is not empty first because create a new ChannelException is expensive
             pendingUnencryptedWrites.removeAndFailAll(new ChannelException("Pending write on removal of SslHandler"));
         }
+        if (engine instanceof OpenSslEngine) {
```

Comment:
@nmittler also it's in the modification section but I can also adjust the commit message to make it more explicit if you want too.